mayhem/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM --platform=linux/amd64 ubuntu:20.04 as builder
```

Comment:
Is this in-place modification of the header file here necessary? This seems prone to breakage.